src/main/groovy/net/wasdev/wlp/gradle/plugins/tasks/InstallAppsTask.groovy

```diff
@@ -378,4 +381,22 @@ class InstallAppsTask extends AbstractServerTask {
         }
         return false
     }
+
+    void createApplicationFolders() {
+        File serverDir = getServerDir(project)
+        File appsDirectory = new File(serverDir, 'apps')
+        File dropinsDirectory = new File(serverDir, 'dropins')
+
+        try {
+            if (!appsDirectory.exists()) {
+                appsDirectory.mkdir()
+            }
+
+            if (!dropinsDirectory.exists()) {
+                dropinsDirectory.mkdir()
+            }
+        } catch (Exception e) {
```

Comment:
It's bothering me a bit that we attempt to create both the apps folder and dropins folder when they may only need one of them. Do you think the end user would wonder why we created the extra directory?